CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+0.5.0 (2021-06-25)
+===
+
+GC Plans
+---
+* Added a new plan PageProtect, a plan to help debugging. It allocates each object
+  to a separate page, and protects pages when the pages are released.
+
+API
+---
+* Major changes to the ObjectModel trait: now a binding can specify each per-object
+  metadata used by mmtk-core, whether the metadata resides in header bits provided
+  by the runtime or side tables provided by mmtk-core. For in header metadata, a binding
```

Comment:
probably it's more clear to say '... the VM runtime' rather than '... the runtime', because the runtime could mean mmtk-core's runtime?